src/bundle/Resources/views/themes/admin/ui/error_page/unknown.html.twig

```diff
@@ -12,8 +12,8 @@
         </div>
         <h2 class="error-page__title">{{ 'error.page.default.title'|trans|desc('Something went wrong. Sorry!') }}</h2>
         <div class="error-page__messages">
-            <p class="error-page__message">{{ 'error.page.default.message'|trans|raw|desc('It\'s an error 500, an internal server error. The server was unable to complete your request and it\'s not your fault.') }}</p>
-            <p class="error-page__message">{{ 'error.page.default.actions'|trans|raw|desc('Please contact your eZ Platform Administrator and inform her/him about the time the error occurred, the URL in your browser window and anything related you think may have caused the error.<br/>You can also go back or try to reload the application.') }}
+            <p class="error-page__message">{{ 'error.page.default.message'|trans|raw|desc('Error 500: internal server error. The server was unable to complete your request because of a problem on our side.') }}</p>
```

Comment:
```suggestion
            <p class="error-page__message">{{ 'error.page.default.message'|trans|raw|desc('Error 500: internal server error. The server was unable to complete your request because of an internal problem.') }}</p>
```